src/index.css

```diff
@@ -0,0 +1,91 @@
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400;500;600;700;800&display=swap');
```

Comment:
maybe you should move this to index.html that way, it loads it data in a sequential order or the order you define

Reply:
Hmm.
I've always being using it this way without checking for the order sequence.
I will do that.
Thank you so much for the comments.
I really appreciate.🙏